fair-data-model/cytokine/competency-questions/CQ4.rq

```diff
@@ -0,0 +1,48 @@
+# QC4 - What are the severity score measurements of a patient and which measurement method was used?
+
+PREFIX rdf:     <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs:    <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX xsd:     <http://www.w3.org/2001/XMLSchema#>
+PREFIX owl:     <http://www.w3.org/2002/07/owl#>
+PREFIX dc:     <http://purl.org/dc/terms/>
+PREFIX obo:     <http://purl.obolibrary.org/obo/>
+PREFIX ncit:    <http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#>
+PREFIX sio:     <http://semanticscience.org/resource/>
+PREFIX bco:     <http://purl.org/beat-covid/cytokines-semantic-model.owl#>
+PREFIX om:      <http://www.ontology-of-units-of-measure.org/resource/om-2/>
+
+ASK WHERE {
+    ?person a sio:SIO_000498 ; 
+              sio:SIO_000228 ?person_patient_role . 
+
+    ?person_patient_role a sio:SIO_000715 ;
+                         sio:SIO_000356 ?severity_score_evaluation_process  .
+
+    ?patient_id a bco:clinical_id  ;
+                  obo:IAO_0000219 ?person_patient_role ;
+                  sio:SIO_000300 ?patient_idstring .
+
+    ?severity_score_evaluation_process a obo:NCIT_C25214 ;
+                                         sio:SIO_000229 ?severity_score_answer ;
+                                         dc:conformsTo ?severity_score_measurement_method ;
+									     sio:SIO_000217 ?severity_score_evaluation_process_date . 
+
+    ?severity_score_evaluation_process_date a obo:NCIT_C25164 ;
+                                              dct:date  ?severity_score_evaluation_process_date_value .
```

Comment:
Prefix `dct` is not declared. The query will throw an error.

Reply:
done